### DISCUSSION TITLE
Is Github going to charge for using Codespace for user?

### DISCUSSION THREAD

**Discussion by atulkamble** [community/community#18272] [Codespaces]
As Codespace is in beta right now, so in future whenever we are using the CodeSpace are we going to get bill for it?
Any idea ?

**Comment by Tolfx** [comment]
As long you are in the beta program you won't be charged.
But once `Codespace` hits production and you'll leave beta access and you will be notified via email before billing begins, so you can adjust your usage.

You can read more on [`Github Docs`](https://docs.github.com/en/billing/managing-billing-for-github-codespaces/about-billing-for-codespaces) about billing for `Codespace`
